src/AdornedRulerPanel.cpp

```diff
@@ -1213,9 +1214,20 @@ void AdornedRulerPanel::OnPaint(wxPaintEvent & WXUNUSED(evt))
 
    if (!mViewInfo->selectedRegion.isPoint())
    {
+#if 0
+      mPlaybackLooping.DrawLoopingRange(&backDC, mInner,
+          max(1, Time2Pos(mViewInfo->selectedRegion.t0())),
+          min(mInner.width, Time2Pos(mViewInfo->selectedRegion.t1()))
+      );
+#endif
       DoDrawSelection(&backDC);
    }
 
+   if (PlaybackLooping::GetInstance().IsProcessing())
+   {
+       mPlaybackLoopingCell->DrawLoopingRange(&backDC, viewInfo);
+   }
+
```

Comment:
TrackPanel uses CellularPanel::Draw which leaves the painting to the Draw() overrides of cell and handle objects.  It would be preferable but perhaps not needed to rewrite AdornedRulerPanel to be like it.